sale_invoicing_date_selection/tests/test_sale_invoicing_date_selection.py

```diff
@@ -48,10 +48,9 @@ def test_andvance_invoice(self):
             )
         ) as wiz_invoice_form:
             wiz_invoice_form.invoice_date = "2024-03-01"
+            wiz_invoice_form.advance_payment_method = "fixed"
+            wiz_invoice_form.fixed_amount = 10
         wiz = wiz_invoice_form.save()
-        # set values by code as there are invisible fields
```

Comment:
I'd say visibility conditions did not change from V16 to V17, why change this?

Reply:
When using just one sale.order record, as in this case, those fields are not invisible.
Also, when having:
```
# set values by code as there are invisible fields
wiz.advance_payment_method = "fixed"
wiz.fixed_amount = 10
```
it returns an error during testing:
```
raise UserError(_('The value of the down payment amount must be positive.'))
odoo.exceptions.UserError: El valor del importe por adelantado debe ser positivo.
```
Even trying `save()` after setting those values
So it wasn't making any effect.